src/microbe_stage/PatchMap.cs

```diff
@@ -259,6 +259,17 @@ public List<Species> FindAllSpeciesWithPopulation()
         return found.ToList();
     }
 
+    /// <summary>
+    ///   Updates the time period in all of the patches.
+    /// </summary>
+    public void UpdateGlobalTimePeriod(double time)
```

Comment:
Why do the history store the time period if all of them are updated here?

Reply:
This is just for updating the current time period in all patches. The history only stores a copy of the time period of when it is recorded. It will then be used in the charts to plot the x axis.

Reply:
Ah okay, so it goes to the current time there. Gotcha. This could be a bit less confusing for me if the idea about the proxy properties is done so that code outside Patch can access the current conditions data without referring to the intermediate conditions object.